### PR TITLE
Gui improvements

### PR DESCRIPTION
**Pull request by STPv22**
forgor abt a screenshot. 

This could use some improvements, such as a better splash animation. The code is also not completely optimal, but I was too lazy to fix it. 
@OeildeLynx31, passing this on to you, ig.

### CODE REVIEW

**Comment by STPv22** [comment]
image

File: modgui.injector.js [hunk 1]
```diff
@@ -1,168 +1,214 @@
...
     "Only causes death 90% of the time!",
     "HTML is better.",
-    "https://github.com/EaglerForge",
     "hey you should check out https://github.com/ZXMushroom63/scratch-gui",
     "99% of people stop gambling before they win big.",
     "Now with free estradiol!",
-    "Now with H.I.V (Hyper Injected Virtual-debugger)"
+    "Now with H.I.V (Hyper Injected Virtual-debugger)",
+    "You like eaglers, don't you ≽^•⩊•^≼",
+    "9/11 victims are the fastest readers! They can go through 110 stories in a few seconds.",
```
Comment (OtterCodes101) [inline modgui.injector.js:11]: 😭

Comment (STPv22) [reply modgui.injector.js:11]: I couldn't resist :3

Comment (STPv22) [reply modgui.injector.js:11]: I might ass a few of these ngl https://github.com/STPv22/WildSplashTexts/blob/main/splashes.txt

**Comment by ZXMushroom63** [comment]
@STPv22 
Can you make sure the GUI accounts for all metadata?
Titles, descriptions, icons, credits and version codes are all optional.

**Comment by STPv22** [comment]
👍

**Comment by STPv22** [comment]
image
file name wont display for jenny_skin for whatever reason
image
hows dis?

**Comment by STPv22** [comment]
image

**Comment by STPv22** [comment]
apparently radman doesn't like black things, so im gonna make sum different GUI's and let you seel how they look.

**Pull request closed without merging**